docs/docs/api/06-Configuration/askui-ui-control-client.md

```diff
@@ -0,0 +1,80 @@
+---
+custom_edit_url: null
+---
+# askui UI Control Client
+
+## Table of Contents
+
+  - [Properties](#properties)
+    - [uiControllerUrl](#uiControllerUrl)
+    - [inferenceClientUrl](#inferenceClientUrl)
+    - [annotationLevel](#annotationLevel)
+    - [credentials](#credentials)
+  - [Set Log Level](#set-log-level)  
+  - [Example](#example)  
+## Properties
+
+### uiControllerUrl
+
+• **uiControllerUrl**: `string` - Default: `http://localhost:6769`
+
+This provides the url where the askui UI Controller runs. The askui UI Control Client is also connecting
+to this address.
+___
+
+### inferenceClientUrl
+
+• **inferenceClientUrl**: `string` - Default: `https://inference.askui.com`
+
+The address of the askui Inference server.
+___
+
+### annotationLevel
+
+• **annotationLevel**: `AnnotationLevel` - Default: `AnnotationLevel.DISABLED`
+
+AnnotationLevel is implemented as an enum. You have three options: `DISABLED`, `ON_FAILURE`, `ALL`.
+
+`ALL`:  Runs the [annotate](../../general/05-Tooling/annotate-image.md) command after the execution of each test step.
+
+
+`ON_FAILURE`: Runs the annotate command if the test step fails 
+
+
+`DISABLED`: Never runs the annotate command after test steps
+___
+
+### credentials
+
+• **credentials**: `string` - Default: `environment-tenant`, `environment-token`, `environment-email`
```

Comment:
There is not default value for `credentials` as far as I know. It is optional.

Also it is no `string` from my perspective but rather an `object`. I would document the name of the properties of that object and their types. Maybe, we can do this like jsdoc does it, e.g., esee https://jsdoc.app/tags-type.html#:~:text=An%20object%20called%20'myObj'%20with,c'%20(any%20type).&text=JSDoc%20supports%20Closure%20Compiler's%20syntax,is%20contained%20in%20the%20array.